Src/TransmitData.c

```diff
@@ -30,7 +34,7 @@ static const int8_t LOWER_VALVE_STATUS_HEADER_BYTE = 0x39;
 #define FLAGS_AND_CRC_SIZE (6) //1 byte for header flag, 1 byte for ender flag, 4 bytes for crc
 #define IMU_SERIAL_MSG_SIZE (37)
 #define BAROMETER_SERIAL_MSG_SIZE (9)
-#define GPS_SERIAL_MSG_SIZE (17)
+#define GPS_SERIAL_MSG_SIZE (25)
```

Comment:
Just wondering why this was changed from 17 to 25?

Reply:
We put 17 there are a place holder because we didn't actually know how large the GPS data would actually be before we got the GPS working.
With the new communication protocol, we need 25 Bytes for the message (header byte + actual message).
- Byte 1: Header Byte (1 B)
- Bytes 2-5: gps_time (4 B)
- Bytes 6-9: latitude_degrees (4 B)
- Bytes 10-13: latitude_minutes (4 B)
- Bytes 14-17: longitude_degrees (4 B)
- Bytes 18-21: longitude_minutes (4 B)
- Bytes 22-25: altitude (4 B long)